### PR TITLE
Additional document upload

### PR DESCRIPTION
**Pull request by cehsu**
These changes set up the additional document upload page.
screen shot 2018-03-21 at 4 53 52 pm

### CODE REVIEW

File: src/js/disability-benefits/526EZ/config/form.js [hunk 1]
```diff
@@ -296,9 +327,306 @@ const formConfig = {
...
+                            type: 'string',
+                            'enum': [
+                              '1',
+                              '2',
+                              '3',
+                              // '4',
+                              '5',
+                              '6'
+                            ],
+                            enumNames: [
```
Comment (cehsu) [inline src/js/disability-benefits/526EZ/config/form.js]: Kindly note that these labels are placeholders, as the final labels are TBD.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu @goldenmeanie @cvalarida This screenshot doesn't show the most up-to-date copy that I provided to Alex for the prototype. 
This is the updated copy: 
If you have other evidence, like lay or buddy statements, that you would like to submit, you can upload them here.

Button should read: Add Another Document

**Comment by cehsu** [comment]
Thanks for the updated content @peggygannon, updated (kindly refer to the updated screenshot above).

**Review by peggygannon — APPROVED** [review]
@cehsu Looks good. thanks for updating

File: src/js/disability-benefits/526EZ/config/form.js [hunk 2]
```diff
@@ -426,11 +431,200 @@ const formConfig = {
...
+                }
+              }
+            }
+          }
+        },
+        documentUpload: {
+          title: 'Lay statements or other evidence',
+          depends: (formData, index) => {
+            const hasOtherEvidence = _.get(`disabilities.${index}.view:otherEvidence`, formData);
+            return hasOtherEvidence;
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/config/form.js]: If we're not going to perform any other operations on this, can we use a "concise body" arrow function? (MDN tells me that's what they're called, anyhow.)
```js
depends: (formData, index) => _.get(...);
```

File: src/js/disability-benefits/526EZ/config/form.js [hunk 3]
```diff
@@ -426,11 +431,200 @@ const formConfig = {
...
+          uiSchema: {
+            disabilities: {
+              items: {
+                additionalDocuments: Object.assign({},
+                  fileUploadUI('Lay statements or other evidence', {
+                    allowRename: true,
+                    itemDescription: 'Adding additional evidence:',
+                    endpoint: '/v0/preneeds/preneed_attachments', // TODO: update this with correct endpoint (e.g. '/v0/21-526EZ/medical_records')
+                    addAnotherLabel: 'Add Another Document',
+                    fileTypes: ['pdf', 'jpg', 'jpeg', 'png'],
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/config/form.js]: Are these TBD or are we sure these are the ones?

File: src/js/disability-benefits/526EZ/helpers.jsx [hunk 4]
```diff
@@ -181,3 +181,31 @@ export const recordReleaseWarning = (
...
+export const additionalDocumentDescription = () => {
+  return (
+    <div>
+      <p>If you have other evidence, like lay or buddy statements, that you would like to submit, you can upload them here.</p>
+      <p>File upload guidelines:</p>
+      <ul>
+        <li>File types you can upload: .pdf, .jpeg, or .png</li>
+        <li>Maximum file size: 50 MB</li>
+      </ul>
+      <p><em>Large files can be more difficult to upload with a slow Internet connection</em></p>
```
Comment (cvalarida) [inline src/js/disability-benefits/526EZ/helpers.jsx]: Should this end with a `.`?

**Review by cvalarida — APPROVED** [review]
Looks great! I had a few small comments, but those aren't blocking. It looks like the parent branch has an additional test for `FileField`, so I'm...hesitant to ask for that to be merged in before merging this branch into master, but I do think we need that test.

If you think it's safer to merge this into master and then pull the test in another branch, I'm okay with that, I think.

:crossed_fingers:

File: src/js/disability-benefits/526EZ/config/form.js (hunk 3 above)
Comment (cehsu) [reply src/js/disability-benefits/526EZ/config/form.js]: Good question, these are in the mock, but I'll include a comment to follow up on this.

File: src/js/disability-benefits/526EZ/config/form.js (hunk 2 above)
Comment (cehsu) [reply src/js/disability-benefits/526EZ/config/form.js]: Good idea. Will change in the next PR.

File: src/js/disability-benefits/526EZ/helpers.jsx (hunk 4 above)
Comment (cehsu) [reply src/js/disability-benefits/526EZ/helpers.jsx]: Good question for @peggygannon. Peggy should this sentence include a period at the end?
*Large files can be more difficult to upload with a slow Internet connection*
screen shot 2018-03-21 at 4 53 52 pm

**Pull request merged**